test/units.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+go get github.com/weaveworks/weave/testing/cover
```

Comment:
My understanding is that go get will use the local code, and install the
binary in you bin dir.

On Saturday, 11 July 2015, Matthias Radestock <notifications@github.com>
wrote:

> In test/units.sh
> <https://github.com/weaveworks/weave/pull/1115#discussion_r34406791>:
>
> > @@ -0,0 +1,28 @@
> > +#!/bin/bash
> > +
> > +go get github.com/weaveworks/weave/testing/cover
>
> It seems very strange to refer to the weave gh repo here. Why can't this
> refer to the local version?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/weaveworks/weave/pull/1115/files#r34406791>.
>